### PR TITLE
[WIP] Add elixir client

### PR DESCRIPTION
**Pull request by niku**
### PR checklist

- [x] Read the [contribution guildelines](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell/batch script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates)
- [x] Filed the PR against the correct branch: master for non-breaking changes and `2.3.0` branch for breaking (non-backward compatible) changes.

### Description of the PR

Add elixir client(alpha)

Related https://github.com/niku/swagger-codegen-for-elixirclient/issues/3

- [x] I think this client generator is alpha version, how to notify this status? 
- [ ] How to write `./bin/security/...-petstore.sh` ?

### CODE REVIEW

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ElixirClientCodegen.java [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    }
+
+    /**
+     * Returns human-friendly help for the generator.  Provide the consumer with help
+     * tips, parameters here
+     *
+     * @return A string value for the help message
+     */
+    public String getHelp() {
+        return "Generates an elixir client library.";
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ElixirClientCodegen.java]: @niku I suggest `Generates an elixir client library (alpha).` to indicate the API lib is still alpha.

Comment (niku) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ElixirClientCodegen.java]: Thank you.
I have rewrited and force pushed.

Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ElixirClientCodegen.java]: @niku thanks. I'll merge it into master shortly.

Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/ElixirClientCodegen.java]: > How to write ./bin/security/...-petstore.sh

I'll take care of it. No worry :)

**Comment by wing328** [comment]
Tweet: https://twitter.com/wing328/status/826007206587817984

**Pull request merged**